### ISSUE TITLE
Question about cquery linter

### ISSUE THREAD

**Issue by gagbo** [w0rp/ale#1635]
Ping @zed0 since you're the one responsible for saving my life with the initializationOptions.

In `ale_linters/cpp/cquery.vim` you pass a double `:h:h` to fnamemodify in order to [find the project root](https://github.com/w0rp/ale/blob/master/ale_linters/cpp/cquery.vim#L10).

Since cquery expects `compile_commands.json` to be at the root of the project, shouldn't the fnamemodify flag be a simple `:h`, so it transforms `/path/to/project/compile_commands.json` into `/path/to/project`. The double `:h:h` would give `/path/to` and I had wrong errors showing in my vim because of this (basically cquery could not find any symbol).

If you made it work with `:h:h` I'd like to see how you placed your `compile_commands.json` in your project because I was not able to make it work.

**Comment by zed0** [comment]
Yeah, you're right, that's embarrassing.  I've discovered that I had a random old copy of `compile_commands.json` lying around in my home directory that it was looking at.  I've opened a pull request to fix this: #1636
